src/themes/base.scss

```diff
@@ -322,12 +322,9 @@ inclusive-dates *:focus-visible {
 }
 
 .inclusive-dates-calendar__keyboard-hint {
-  color: var(--secondary-color);
   display: flex;
```

Comment:
The `<summary>` icon does not align. 
<img width="476" alt="image" src="https://github.com/user-attachments/assets/0074e405-8fe1-474d-b368-0751f27c6c0e">

But we might do something like this:

```
    display: inline-flex;
    transform: translateY(2px);
```